spec/features/deploy_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'spec_helper'
 require 'support/aruba_helper'
 
+# TODO: These tests are failing when using the aruba git setup
+# expected that command "git push" has exit status of "0", but has "128".
```

Comment:
These tests are failing, but I haven't been able to get to the bottom of why yet.

Reply:
Fixed by James